pegasus/sites.v3/code.org/public/v3/edit-csp-table/splat.haml

```diff
@@ -103,6 +104,12 @@ angular: recordsApp
     }
     getMetadata(true);
 
+    $scope.selectedFile = function() {
+      $scope.$apply(function () {
+        $scope.fileSelected = true;
+      });
+    }
```

Comment:
actually you can probably scrap this function and just do `fileSelected = true` instead of `selectedFile()` above